saleor/graphql/payment/mutations/payment/checkout_payment_create.py

```diff
@@ -213,6 +213,10 @@ def perform_mutation(  # type: ignore[override]
     ):
         checkout = get_checkout(cls, info, checkout_id=checkout_id, token=token, id=id)
 
+        use_legacy_error_flow_for_checkout = (
+            checkout.channel.use_legacy_error_flow_for_checkout
```

Comment:
In this place, we execute additional DB query for channel.

Reply:
https://github.com/saleor/saleor/pull/13350#discussion_r1266360290